admin/broadleaf-contentmanagement-module/pom.xml

```diff
@@ -109,5 +109,16 @@
             <artifactId>spring-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
This is defined in the top level pom line 689. Can this be removed here and Use top level pom to exclude?